department-of-veterans-affairs/next-build
# 15056 breadcrumb component

## Description
Relates to #[15056](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15056). (or closes?)

Changes:
- Adds breadcrumb component to next-builld, mimicking the functionality in content-build but using the VA design system breadcrumbs web-component.
- Modifies queries to include breadcrumbs in resource objects where needed
- Adds unit tests for breadcrumb component
- Adds unit tests for breadcrumb utility functions
- Adds storyboard component
- Adds the component to newsStory and storyListing layouts



## Testing done
Local visual testing

## Screenshots
<img width="1181" alt="image" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/81648012-4cc0-4eea-b6e5-62b23381ee82">


## QA steps
- verify that breadcrumbs render on appropriate pages
- verify styles match prod

## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:
